composer.lock

```diff
@@ -288,16 +288,16 @@
         },
         {
             "name": "nikic/php-parser",
-            "version": "v4.10.2",
+            "version": "v4.10.3",
```

Comment:
this change will break some our tests. I've already fixed our tests but didn't push code.

Reply:
fixed in https://github.com/infection/infection/pull/1454